uefi-raw/src/protocol/misc.rs

```diff
@@ -22,3 +23,26 @@ pub struct TimestampProperties {
     /// example, a 24-bit counter would have an end value of `0xff_ffff`.
     pub end_value: u64,
 }
+
+/// Properties of Reset Notification.
+#[derive(Debug)]
+#[repr(C)]
+pub struct ResetNotificationProtocol {
+    pub register_reset_notify:
+        unsafe extern "efiapi" fn(this: *mut Self, reset_function: Option<ResetSystemFn>) -> Status,
+    pub unregister_reset_notify:
+        unsafe extern "efiapi" fn(this: *mut Self, reset_function: Option<ResetSystemFn>) -> Status,
+}
+
+impl ResetNotificationProtocol {
+    pub const GUID: Guid = guid!("9da34ae0-eaf9-4bbf-8ec3-fd60226c44be");
+}
+
+/// Raw reset notification function, to be called if you register it when a RestSystem() is executed.
+// similar to uefi-raw/src/table/runtime.rs:53  at commit@6093205c3eb27b2e78be4c003c04d46679bff420
```

Comment:
nit: drop the `// similar to ...` comment, it's not telling us anything very important